invenio_records_resources/services/files/config.py

```diff
@@ -39,10 +39,14 @@ class FileServiceConfig(ServiceConfig):
 
     max_files_count = 100
 
-    # Inheriting resource config should define these
+    # Inheriting service config should define these
     file_links_list = {}
     file_links_item = {}
 
+    # At the resource level and link serialization (service) level
```

Comment:
Have not touched any of the services to enforce this, since the previous functionality didn't do this either.